shoreline/manifest.json

```diff
@@ -36,7 +36,8 @@
         "caption": "Example of fleetwide linux command details",
         "image_url": "images/fleetwide_linux_command_details.png"
       }
-    ]
+    ],
+    "changelog": ""
```

Comment:
why are you adding this here?

Reply:
I added this because of the following error, but I saw that @mgarabed already fixed this on 04/19/2022. I will remove the key because is duplicate.

<img width="1409" alt="image" src="https://user-images.githubusercontent.com/85694931/166204600-724a936a-8322-4058-a2b9-f6723caba358.png">